_docs/_latest/setup/migrate.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Migrate from Python Installer to Terraform Module
+order: 007
+---
+
+# {{ page.title }}
+
+This guide explains how to migrate a Forseti deployment from the
+deprecated Python Installer to the new Terraform module.
+
+---
+
+## Before You Begin
+
+Before you begin the migration process, you will need:
+
+- A version of the
+  [Terraform command-line interface](https://www.terraform.io/downloads.html)
+  in the 0.12 series
+- The domain and ID of the Google Cloud Platform (GCP) organization in
+  which Forseti is deployed.
+- The ID of the GCP project in which Forseti is deployed.
+- The suffix appended to the names of the Forseti resources; this is
+  likely a 7 digit number.
+- A
+  [key file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys)
+  for a service account in the organization with the
+  [roles required by the Terraform module](https://registry.terraform.io/modules/terraform-google-modules/forseti/google/4.1.0#iam-roles).
+
+## Configuring Terraform
+
+In the working directory, create a file named `main.tf` with content
+like the following, replacing the first four uppercased values with the
+aforementioned values:
+
+```hcl
+terraform {
+  required_version = ">= 0.12.0"
+}
+
+module "forseti" {
+  source = "terraform-google-modules/forseti/google"
+  version = "~> 4.1"
```

Comment:
Update the version to 4.2, this will be the version of the next Terraform release.

Reply:
Fixed in 8f24b3723384a0707cf11dc88b4f4c4256b96928.